patroni/api.py

```diff
@@ -1263,6 +1263,7 @@ def get_postgresql_status(self, retry: bool = False) -> Dict[str, Any]:
             row = self.query(stmt.format(postgresql.wal_name, postgresql.lsn_name,
                                          postgresql.wal_flush), retry=retry)[0]
             result = {
+                'name': postgresql.name,
```

Comment:
It doesn't deserve a separate PR.
```suggestion
                'scope': postgresql.scope,
                'name': postgresql.name,
```